docs/book/customers/customer_and_shopuser.rst

```diff
@@ -77,6 +77,40 @@ The already set password of a **ShopUser** can be easily changed via the ``setPl
    $user->setPlainPassword('resu1');
    // the password will now be 'resu1' and will become encrypted while saving the user in the database
 
+
+Customer related events
+-----------------------
+
++---------------------------------------------+-----------------------------------------------------------------------------------------+
+| Event id                                    | Description                                                                             |
```

Comment:
`Description` column actually says what listeners do on that event, I was expecting to get some information when and where that event happens.